### PR TITLE
docs: update bisect instructions

### PR DESCRIPTION
**Pull request by aanm**
The current documentation was not working. With these small adjustments they are working again.

### CODE REVIEW

File: Documentation/contributing/testing/e2e_legacy.rst [hunk 1]
```diff
@@ -70,8 +70,8 @@ usage information.
 
    .. code-block:: shell-session
 
-      $ HELM_VERSION=3.7.0
-      $ wget "https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz"
+      $ HELM_VERSION=v3.13.1
+      $ wget "https://get.helm.sh/helm-${HELM_VERSION}-linux-amd64.tar.gz"
       $ tar -xf "helm-v${HELM_VERSION}-linux-amd64.tar.gz"
```
Comment (ghost) [inline Documentation/contributing/testing/e2e_legacy.rst:75]:
```suggestion
      $ tar -xf "helm-${HELM_VERSION}-linux-amd64.tar.gz"
```
there is now a double `v` here

Comment (aanm) [reply Documentation/contributing/testing/e2e_legacy.rst:75]: No, I've removed it

**Review by bimmlerd — APPROVED** [review]
No idea how this works, but change looks sane

**Pull request merged**